
export const addEmployee = (req, res, next) => {
    try {

    } catch (error) {
        next(error)
    }
}